PonyEdit.pro

```diff
@@ -61,11 +51,14 @@ macx {
     QMAKE_POST_LINK += /usr/bin/install_name_tool -change /usr/local/lib/libssh2.1.dylib @executable_path/libssh2.1.dylib $$OUT_PWD/PonyEdit.app/Contents/MacOS/PonyEdit;
 	QMAKE_POST_LINK += /usr/bin/install_name_tool -change /usr/local/ssl/lib/libssl.1.0.0.dylib @executable_path/libssl.1.0.0.dylib $$OUT_PWD/PonyEdit.app/Contents/MacOS/PonyEdit;
 	QMAKE_POST_LINK += /usr/bin/install_name_tool -change /usr/local/ssl/lib/libcrypto.1.0.0.dylib @executable_path/libcrypto.1.0.0.dylib $$OUT_PWD/PonyEdit.app/Contents/MacOS/PonyEdit;
+}
+
+linux|macx {
+    TARGET = PonyEdit
 
-	QMAKE_CFLAGS += -Werror -Wunused-parameter
-	QMAKE_CXXFLAGS += -Werror -Wunused-parameter
+    LIBS += -lz -lssh2 -lcrypto -lssl
 
-    LIBS	+= -lssh2 -lcrypto -lssl
+    QMAKE_CXXFLAGS += -Werror -Wunused-parameter -Weffc++ -include $$PWD/main/global.h
```

Comment:
I assume this is because Qt Creator has to be told about where to find include files, in addition to telling GCC where to find them at compile time? 

It'd be nicer if Qt Creator was smarter, but if it's not, there isn't much we can do about it.